index.js

```diff
@@ -28,7 +28,14 @@ const exampleMovies = require("./movies");
       "James and the Giant Peach",
     ];
  */
-function getAllMovieTitles() {}
+function getAllMovieTitles(movies) {
+  let newArray = []; 
+  for (let i = 0; i < movies.length; i++) {
```

Comment:
Since we not using `i` for anything other than accessing an element of the `movies` array, we should use a `for...of` loop here.